docs/components/board/_index.md

```diff
@@ -1115,7 +1060,7 @@ For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/r
 
 ### Read
 
-Read the current integer value of the digital signal output by the [ADC](#analogs).
+Read the current value from the analog reader.
```

Comment:
We should update the return of that analog Read function it now returns the following message in value: https://github.com/viamrobotics/api/blob/4f9d14d067b9b7965fcb4e89d6185e9dc348c3cd/proto/viam/component/board/v1/board.proto#L191

So it is not a struct, not one integer,  the sturct returns are as value.Value (integer), value.Min (float) value.Max (float), value.StepSize (float)

Reply:
Ok thanks!